<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VPAWModelOCT</class>
 <widget class="qMRMLWidget" name="VPAWModelOCT">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>286</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="VPAWModelButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Go to VPAW Model module</string>
     </property>
     <property name="text">
      <string>Go to VPAW Model module</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="VPAWVisualizeButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Go to VPAW Visualize module</string>
     </property>
     <property name="text">
      <string>Go to VPAW Visualize module</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="HomeButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Go to Home module</string>
     </property>
     <property name="text">
      <string>Go to Home module</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="VPAWVisualizeOCTButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Go to VPAW Visualize OCT module</string>
     </property>
     <property name="text">
      <string>Go to VPAW Visualize OCT module</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="linkOCTSegCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>VPAW Model OCT: Link to OCT Segmentation Codebase</string>
     </property>
     <property name="collapsed" stdset="0">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="linkOCTSegFormLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="octSegDirectoryLabel">
        <property name="text">
         <string>OCT Segmentation source directory</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ctkPathLineEdit" name="OCTSegDirectory">
        <property name="filters">
         <set>ctkPathLineEdit::Dirs</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="linkOCTSegButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Link to OCT Segmentation codebase and install dependencies</string>
        </property>
        <property name="text">
         <string>Link to OCT Segmentation codebase and install dependencies</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="runOCTSegCollapsibleButton">
     <property name="text">
      <string>VPAW Model OCT: Run OCT Segmentation</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="VPAWOCTConfigFileLabel">
        <property name="text">
         <string>YAML Configuration File</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ctkPathLineEdit" name="VPAWOCTConfigFile">
        <property name="filters">
         <set>ctkPathLineEdit::Files</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="runOCTSegButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Run OCT Segmentation</string>
        </property>
        <property name="text">
         <string>Run OCT Segmentation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
